front-end/build.gradle

```diff
@@ -18,8 +18,8 @@ plugins {
 }
 
 node {
-    version = '10.14.1'
-    npmVersion = '6.4.1'
+    version = '16.19.1'
```

Comment:
Or use the 18.16 LTS version?

Reply:
I may do it after 0.4.0 released or we find further blocker to fix :D